src/CarbonAware.WebApi/src/Program.cs

```diff
@@ -28,6 +28,11 @@
     c.EnableAnnotations();
     c.OperationFilter<CarbonAwareParametersBaseDtoOperationFilter>();
     c.SchemaFilter<CarbonAwareParametersBaseDtoSchemaFilter>();
+    c.SwaggerDoc("v1", new OpenApiInfo
+    {
+        Version = "CarbonAware.WebAPI",
+        Title = "1.0.0",
```

Comment:
Should this be added to project assembly variables in the future? so version is not maintained as hardcoded value in the code?

Reply:
Let's apply the related changes (if any, in the NPM PR that should follow)